contents/docs/feature-flags/manual.mdx

```diff
@@ -1,159 +0,0 @@
-    // Do something differently for this user
-}
-```
-
-```php
-if (PostHog::isFeatureEnabled('new-beta-feature', 'user distinct id')) {
-    // do something here
-}
-```
-
-```ruby
-is_my_flag_enabled = posthog.is_feature_enabled('new-beta-feature', 'user distinct id')
-
-if is_my_flag_enabled
-  # Do something differently for this user
-end
-```
-
-```go
-isFlagEnabledForUser, err := client.IsFeatureEnabled(
-    FeatureFlagPayload{
-        Key:        "new-beta-feature",
-        DistinctId: "distinct-id",
-    })
-
-if (isFlagEnabledForUser) {
-  // Do something differently for this user
-}
-```
-
-```python
-if posthog.feature_enabled("new-beta-feature", "user_distinct_id"):
-    runAwesomeFeature()
-```
-
-```
-curl https://app.posthog.com/decide/ \
--X POST -H 'Content-Type: application/json' \
--d '{
-    "api_key": "phc_Wg2jMXmDvsnrQ3IKuVyspZghFIVE2IbxuqVYlhm7ry0",
-    "distinct_id": "[user distinct id]"
-}'
-```
-
-</MultiLanguage>
-
-What you do inside that if statement is up to you. You might change the CSS of a button, hide an entire section, or move elements around on the page.
-
-### Ensuring flags are loaded before usage
-
-Every time a user loads a page we send a request in the background to an endpoint to get the feature flags that apply to that user. In the client, we store those flags as a cookie.
-
-This means that for most page views the feature flags will be available immediately, _except_ for the first time a user visits.
-
-To combat that, there's a JavaScript callback you can use to wait for the flags to come in:
-
-```js-web
-posthog.onFeatureFlags(function () {
-    // feature flags are guaranteed to be available at this point
-    if (posthog.isFeatureEnabled('new-beta-feature')) {
-        // do something
-    }
-})
-```
-
-## Persisting feature flags across authentication steps
-
-You have an option to persist flags across authentication steps.
-
-Consider this case: An anonymous person comes to your website and you use a flag to show them a green call to action.
-
-Without persisting feature flags, the flag value can change on login because their identity can change (from anonymous to identified). Once they login, the flag might evaluate differently and show a red call to action instead.
-
-This usually is not a problem since experiments run either completely for anonymous users, or completely for logged in users.
-
-However, with some businesses, like e-commerce, it's very common to browse things anonymously and login right before checking out. In cases like these you can preserve the feature flag values by checking this checkbox.
-
-![Persist feature flags](../../images/features/feature-flags/feature-flag-persist.png)
-
-Note that there are some performance trade-offs here. Specifically,
-
-1. Enabling this slows down the feature flag response.
-2. It disables local evaluation of the feature flag.
-3. It disables bootstrapping this feature flag.
-
-## Feature flags versus experiments
```

Comment:
Could we add this to common questions?

Reply:
I dont think this is needed in the feature flag docs. In the experiment docs, this is mentioned here -> https://posthog.com/docs/experiments/creating-an-experiment#feature-flag-key